controllers/keycloak.go

```diff
@@ -11,6 +13,38 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+func keycloakConfigMap(cr *hyperfoilv1alpha1.Horreum) *corev1.ConfigMap {
+	req, err := http.NewRequest("GET", "https://raw.githubusercontent.com/Hyperfoil/Horreum/0.7.11/infra/keycloak-horreum.json", nil)
+	if err != nil {
+		return nil
+	}
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil
+	}
+
+	defer resp.Body.Close()
+
+	horreumRealm, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil
+	}
+
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cr.Name + "-keycloak-horreum",
```

Comment:
I find this name non-descriptive / confusing. I suggest `cr.Name + "-keycloak-horreum-realm"`

Reply:
Done